actions/drain_node_handler.go

```diff
@@ -12,24 +12,35 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sync/errgroup"
 	v1 "k8s.io/api/core/v1"
+	policyv1 "k8s.io/api/policy/v1"
 	"k8s.io/api/policy/v1beta1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubectl/pkg/drain"
 
 	"github.com/castai/cluster-controller/castai"
 )
 
+const (
+	minDrainTimeout   = 60               // Minimal pod drain timeout
```

Comment:
This might be too big if controller is being restarted every 30seconds. This could be much shorter if we'd know that we already sent evict some time ago. As it's async operation.